libbeat/scripts/kibana_import.py

```diff
@@ -0,0 +1,47 @@
+from elasticsearch import Elasticsearch
+import argparse
+import os
+
+def store_object(es, type, name, doc):
+    print es.index(index=".kibana", doc_type=type, id=name, body=doc)
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Loads Kibana dashboards, vizualization and " +
+                    "searches into Kibana")
+    parser.add_argument("--url", help="Elasticsearch URL. E.g. " +
+                                      "http://localhost:9200.", required=True)
+    parser.add_argument("--dir", help="Input directory (kibana folder)", default="saved", required=True)
```

Comment:
It would be nice to have the same extra options for the kibana_import script as well.